docs/source/quark_script.rst

```diff
@@ -2081,3 +2081,82 @@ Quark Script Result
     $ python3 CWE-78.py
     CWE-78 is detected in method, Lcom/vuldroid/application/RootDetection; onCreate (Landroid/os/Bundle;)V
 
+
+
+Detect CWE-117 in Android Application (allsafe.apk)
+------------------------------------------------------
+This scenario seeks to find output doesn’t neutralize or incorrectly neutralizes into the Log file. See `CWE-117 <https://cwe.mitre.org/data/definitions/117.html>`_ for more details.
+
+Let’s use this `APK <https://github.com/t0thkr1s/allsafe>`_ and the above APIs to show how the Quark script finds this vulnerability.
+
+First, we design a detection rule ``findUnneutralizationOutput.json`` to spot on behavior using the method dealing with no neutralization secret. 
+
+Then, we get all parameter values that are input to this method. From the returned parameter values, we use some keywords such as escape, replace, format, and getLogger to check if the arguments didn’t neutralize.
+
+If the answer is YES, that may cause secret context leakage into the log file or the attacker might perform a log injection attack using encoded methods, or the attacker might perform a log injection attack using encoded methods.
+
+Quark Script CWE-117.py
+=======================
+
+.. code-block:: python
+
+    from quark.script import runQuarkAnalysis, Rule
+
```

Comment:
The Python code does not meet the [PEP8](https://peps.python.org/pep-0008/)  guide.
I suggest you use [pycodestyle](https://pypi.org/project/pycodestyle/) and [black](https://pypi.org/project/black/) to fix the coding style.